Makefile

```diff
@@ -125,52 +125,68 @@ clear_docker_containers:
+	$(docker_command) ./gradlew -p $(project) $(log_level) $(params) publishToMavenLocal -PprojectVersion=local
 
 stage_ui_tests:
 	make publish_to_maven_local
-	./gradlew $(project) $(log_level) $(params) :android-test:ui-testing-core-app:instrumentationUi -DinfraVersion=local
+	./gradlew -p $(project) $(log_level) $(params) :android-test:ui-testing-core-app:instrumentationUi -DinfraVersion=local
 
 test_runner_instrumentation:
 	./gradlew -p samples :test-runner:instrumentationUi --stacktrace
 
 unit_tests:
-	$(docker_command) ./gradlew $(project) $(log_level) $(params) test
+	$(docker_command) ./gradlew -p $(project) $(log_level) $(params) test
 
 gradle_test:
-	$(docker_command) ./gradlew $(project) $(log_level) $(params) gradleTest
+	$(docker_command) ./gradlew -p $(project) $(log_level) $(params) gradleTest
 
 integration_tests:
-	$(docker_command) ./gradlew $(project) $(log_level) $(params) integrationTest
+	$(docker_command) ./gradlew -p $(project) $(log_level) $(params) integrationTest
 
 compile_tests:
-	$(docker_command) ./gradlew $(project) $(log_level) $(params) compileTestKotlin
+	$(docker_command) ./gradlew -p $(project) $(log_level) $(params) compileTestKotlin
 
 compile:
-	$(docker_command) ./gradlew $(project) $(log_level) $(params) compileAll
+	$(docker_command) ./gradlew -p $(project) $(log_level) $(params) compileAll
 
 check:
-	$(docker_command) ./gradlew $(project) $(log_level) $(params) check
+	$(docker_command) ./gradlew -p $(project) $(log_level) $(params) check
 
 .PHONY: build
 build:
-	$(docker_command) ./gradlew $(project) $(log_level) $(params) build
+	$(docker_command) ./gradlew -p $(project) $(log_level) $(params) build
 
 fast_check:
-	$(docker_command) ./gradlew $(project) $(log_level) $(params) compileAll detektAll test
+	$(docker_command) ./gradlew -p $(project) $(log_level) $(params) compileAll detektAll test
 
 full_check:
-	$(docker_command) ./gradlew $(project) $(log_level) $(params) compileAll detektAll test gradleTest
+	$(docker_command) ./gradlew -p $(project) $(log_level) $(params) compileAll detektAll test gradleTest
 
 clean_fast_check:
 	make clean
-	$(docker_command) ./gradlew $(project) $(log_level) $(params) compileAll detektAll test --rerun-tasks --no-build-cache
+	$(docker_command) ./gradlew -p $(project) $(log_level) $(params) compileAll detektAll test --rerun-tasks --no-build-cache
 
 detekt:
-	$(docker_command) ./gradlew $(project) $(log_level) $(params) detektAll
+	$(docker_command) ./gradlew -p $(project) $(log_level) $(params) detektAll
+
+# Analyze modules dependencies issues
+# https://github.com/autonomousapps/dependency-analysis-android-gradle-plugin/wiki/Tasks#build-health
+build_health:
+	$(docker_command) ./gradlew -p $(project) $(log_level) $(params) buildHealth
+
+# Builds dependencies graph
+# https://github.com/autonomousapps/dependency-analysis-android-gradle-plugin
+# Example: make project_graph_report id=:test-runner:client
```

Comment:
For now, only reverse dependencies for the specific project seem usable.
A full graph is useless because it's huge and can't filter a subgraph.
I'll file a feature request for this.

Example for `:test-runner:test-report-api`:
![Screenshot from 2021-06-03 10-34-33](https://user-images.githubusercontent.com/1104540/120605943-514afd80-c457-11eb-8531-469bac1792ba.png)